frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/BasicFieldPanel.tsx

```diff
@@ -61,5 +98,158 @@ export const BasicFieldPanel = ({ searchValue }: { searchValue: string }) => {
         </Box>
       )}
     </Droppable>
+  ) : (
+    <>
+      <Droppable isDropDisabled droppableId={CREATE_FIELD_FREE_TEXT_DROP_ID}>
+        {(provided) => (
+          <Box ref={provided.innerRef} {...provided.droppableProps}>
+            {filteredCreateBasicFreeTextFields.length > 0 && (
+              <FieldSection label="Free text">
+                {filteredCreateBasicFreeTextFields.map(
+                  ({ fieldType, originalIndex }) => (
+                    <DraggableBasicFieldListOption
+                      index={originalIndex}
+                      isDisabled={isLoading}
+                      key={originalIndex}
+                      fieldType={fieldType}
+                    />
+                  ),
+                )}
+              </FieldSection>
+            )}
+            <Box display="none">{provided.placeholder}</Box>
+          </Box>
+        )}
+      </Droppable>
+      <Droppable
+        isDropDisabled
+        droppableId={CREATE_FIELD_BASIC_FIELDS_OPTIONS_DROP_ID}
+      >
+        {(provided) => (
+          <Box ref={provided.innerRef} {...provided.droppableProps}>
+            {filteredCreateBasicFreeTextFields.length > 0 && (
```

Comment:
Should this be `filteredCreateBasicOptionsFields` instead? 
If I'm reading it right, the this is the section for `filteredCreateBasicOptionsFields`.
```suggestion
            {filteredCreateBasicOptionsFields.length > 0 && (
```

Reply:
Thanks for this catch - updated this